src/lib/merkle_tree.ts

```diff
@@ -1,3 +1,8 @@
+/**
+ * This file contains all code related to the [Merkle Tree](https://en.wikipedia.org/wiki/Merkle_tree) implementation available in SnarkyJS.
+ * @author TODO: should we add authors? Some Name <name@o1labs.org>
```

Comment:
we can probably ignore a dedicated list of author, or wdyt?

Reply:
For most core functionality, I would agree that we could probably ignore it but I like having authors for data structures that are added on top. Things like this and `CircuitString` can add authors and give credit on the people implementing these! Just my thoughts though :D

Reply:
I think we can leave this out.